### PR TITLE
Automate Poison Weapon feat chain

### PR DESCRIPTION
**Pull request by Far2Casual**
- added frequency based on actor level (easy to track usage, reset daily),  used new "selfEffect" element
- fix effect duration to end of next turn
- merged all effect into one

image
image

### CODE REVIEW

**Comment by TikaelSol** [comment]
If we are deleting the old effect items we have to migrate away old links to them.

**Comment by Far2Casual** [comment]
Do you think it's worth it, or should I refrain from that ?

**Comment by TikaelSol** [comment]
We have a big list of effects to condense, if we condense them and remove the links we can add these to that list and just do one big migration when we get around to it instead of deleting the effects now. #4144

File: packs/feat-effects/effect-poison-weapon.json [hunk 1]
```diff
@@ -41,6 +41,37 @@
                 "diceNumber": 1,
                 "dieSize": "d4",
                 "key": "DamageDice",
+                "predicate": [
+                    {
+                        "not": "feat:improved-poison-weapon"
```
Comment (simonward) [inline packs/feat-effects/effect-poison-weapon.json:46]: Could use nor.

**Comment by Far2Casual** [comment]
You can pick any part of this and integrate it in a bigger migration of course.

**Pull request merged**